pkg/server/requestLimiter.go

```diff
@@ -0,0 +1,53 @@
+// Copyright Contributors to the Open Cluster Management project
+
+package server
+
+import (
+	"net/http"
+	"sync"
+	"time"
+
+	klog "k8s.io/klog/v2"
+
+	"github.com/gorilla/mux"
+	"github.com/stolostron/search-indexer/pkg/config"
+)
+
+var requestsProcessing = map[string]time.Time{}
+var lock = sync.RWMutex{}
```

Comment:
can we be more specific with the name of this lock? maybe requestLock?

Reply:
How about `requestTracker` and `requestTrackeLock` ?

Reply:
either sounds good to me.. maybe `requestTrackeLock` to keep consistent with other components lock names

Reply:
changed, thanks!